packages/perseus/src/renderer.tsx

```diff
@@ -132,7 +132,7 @@ export type Widget = {
         path: FocusPath,
         newValue: string,
         // TODO(jeremy): I think this is actually a callback
-        focus: () => unknown,
+        focus?: () => unknown,
```

Comment:
This callback/focus function is truly optional.  The API tests in this PR illustrate that by not passing it sometimes.